### ISSUE TITLE
[Train] feature request for catboost_ray 

### ISSUE THREAD

**Issue by rokrokss** [ray-project/ray#26687]
### Description

we're using xgboost_ray and lightgbm_ray with satisfaction, are there any plans for catboost_ray?

**Comment by richardliaw** [comment]
Hi sorry for slow reply - we are not planning to support catboost any time soon but feel free to make a contribution if you are interested!

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
